target/linux/mediatek/dts/mt7986a-mercusys-mr90x-v1.dts

```diff
@@ -0,0 +1,268 @@
+// SPDX-License-Identifier: (GL-2.0 OR MIT)
+
+/dts-v1/;
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/leds/common.h>
+
+#include "mt7986a.dtsi"
```

Comment:
According to your description, it should be mt7986b here?
Also the file name.

Reply:
Right, it should be `mt7986b`. Done in https://github.com/openwrt/openwrt/commit/e570c3390cce795b398a32d35e0c55eedd739928.